flowzone.yml

```diff
@@ -4086,6 +4086,26 @@ jobs:
           with_backoff aws cloudformation validate-template \
             --template-url "https://s3.amazonaws.com/${{ steps.make_bucket.outputs.s3_bucket }}/${tmpvalid}"
 
+      - name: Prepare (Python) Lambda dependencies
+        run: |
+          source '${{ steps.functions.outputs.with_backoff }}'
```

Comment:
Is this line needed?

Reply:
Good catch, nothing to retry in this step